index.html

```diff
@@ -11,22 +11,186 @@
+                            
+                            <ul class="buttons">
+                            <li><a class="button">Bootstrap</a></li>
+                            <li><a class="button">Ruby on rails</a></li>
+                            <li><a class="button">RSpec</a></li>
+                            <li><a class="button">Capybara</a></li>
+                            <li><a class="button">Selenium</a></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box3">
+                        <div class="skills">
+                            <img src="images/icon-skills.svg" alt="icon skills" class="icon">
+                            <h2 class="skills">Skills</h2>
+                            <ul class="buttons">
+                            <li><a class="button">Codekit</a></li>
+                            <li><a class="button">GitHub</a></li>
+                            <li><a class="button">Codepen</a></li>
+                            <li><a class="button">Gitlab</a></li>
+                            <li><a class="button">Terminal</a></li>
+                        </ul>
+                        </div>        
+                    </div>
+                </div>
+            <br>
+            <br>
+            <br>
+            <div class="aboutyou">
+                <div class="comment"><h2 class="aboutyou-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+                <div class="form"><form action="https://formspree.io/f/mbjwboje" method="post">
+                    <input class="textarea" type="text" placeholder="name" maxlength="30" required>
+                    <input class="textarea" type="email" placeholder="email" required>
+                    <textarea class="textarea" name="mdr_message" id="message" cols="50" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                   <button class="projects" type="submit">Get in touch
+                        </button>
+                </form></div>
+            </div>
+        </section>
+        </body>
+               <footer>
+                <div class="bottom">
+                    <hr>
+                <ul class="icons">
+                    <li><a href="www.twitter.com" ><img src="images/twitter.png" alt="twitter"></a></li>
+                    <li><a href="www.linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="images/medium.png" alt="medium"></a></li>
+                    <li><a href="#"><img src="images/github.png" alt="github"></a></li>
+                    <li><a href="#"><img src="images/angellist.png" alt="angellist"></a></li>    
+                </ul>
+                <a class="mail" href="mariaamaliagomezmoro@gmail.com">mariaamaliagomezmoro@gmail.com</a>
+            </div>
+            </footer>        
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_243](https://user-images.githubusercontent.com/69638013/173065071-a16f8980-2217-4127-8355-997604c07b63.png)  |  ![Screenshot_242](https://user-images.githubusercontent.com/69638013/173064806-3d49ce52-c8ed-4a0b-8a00-b2cc7b88defd.png)
- [ ] Please align the Get in Touch button to the left, please do this as this was requested by the previous reviewer 👍🏿 

### OPTIONAL
- Please delete the margin-bottom after the footer , this will make your design look like the Figma design 👍🏿